packages/babel-preset-env-standalone/README.md

```diff
@@ -0,0 +1,5 @@
+babel-preset-env-standalone
+================
+
+babel-preset-env-standalone is a standalone build of [babel-preset-env](https://github.com/babel/babel-preset-env) for use in non-Node.js environments, including browsers. Epsecially with [babel-standalone](https://github.com/babel/babel-standalone). 
```

Comment:
Maybe rewrite the whole README to match [babili's format](https://github.com/babel/babel-standalone/blob/master/packages/babili-standalone/README.md) ?